openfisca_core/json_to_test.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
```

Comment:
If I understand correctly, you chose to extract *biryani converters* to a dedicated module, which is a good choice! This lightens the code of the Core.

But I'm not 100% satisfied with the name `json_to_test.py`. `test` makes me think about a unit test, here it is a test case.